### ISSUE TITLE
make bot summonable

### ISSUE THREAD

**Issue by benadida** [benadida/teamstatus#13]
so you can just invite it into a channel

**Comment by lloyd** [comment]
the more cool sh*t you build into this, the more it almost feels like yak shaving a bit and breaking out the commonality between ircloggr and teamstatus into a framework might be worthwhile.  probably still premature, but interesting to noodle on.

**Comment by benadida** [comment]
agreed, and in fact I'm basing this on irclogr code, so I'm hoping that the yak shaving can be done incrementally as new features are added.
